gateway/1.4/modules/ROOT/pages/flex-install.adoc

```diff
@@ -1,35 +1,32 @@
+//TODO: IS THIS TITLE CORRECT? 
 = Installing Flex Gateway
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images
 
-
-You can install Flex Gateway as a Linux service, in a Docker container, or as a Kubernetes Ingress controller.
+//TODO: Need to clarify what this step is for in each case:
+//      For k8 and OpenShift, this is a prerequisite to _registering_ Flex. 
+//      It does not install to a k8 cluster.
+//
+//      Q: In the Linux and Docker cases, are we running the Flex that we 
+//         download or just downloading Flex so we can register it? 
+You can install Flex Gateway as a Linux service, in a Docker container, or on a Kubernetes cluster.
 
 * If you want to run your Flex Gateway inside a virtual machine (VM), install
 it as a Linux service. 
 * If you want try the product quickly or demonstrate some of the product's capabilities in a non-production
 scenario, install it in a Docker container.
 * If you want use Flex Gateway as a load balancer at the edge of a Kubernetes cluster,
 install it as a Kubernetes Ingress controller.
+//TODO: is this the only installation use case?
 * If you want use Flex Gateway as a load balancer at the edge of an OpenShift cluster,
 install it as an OpenShift Ingress controller.
 
-[cols="1a,1a,1a,1a"]
-|===
-|image:install-linux-logo.png[20%,20%,xref="#install-linux"]
-|image:install-docker-logo.png[25%,25%,xref="#install-docker"]
-|image:install-kubernetes-logo.png[20%,20%,xref="#install-kubernetes"]
-|image:install-openshift-logo.png[20%,20%,xref="#install-openshift"]
-
-|<<install-linux,Linux>>
-|<<install-docker,Docker>>
-|<<install-kubernetes,Kubernetes>>
-|<<install-openshift,OpenShift>>
-|===
+//table with tech logos and links (linux, docker, k8, openshift)
```

Comment:
@chris-campbell9 Hey, While adding OpenShift, I converted the table to an include. Changing the anchors here and using same anchors in the reg-run pages so that the table will work everywhere. The anchors elsewhere with page title syntax were fragile anyway, so now we can change those titles if need be without issues.